content/en/aws/managed-streaming-for-kafka/index.md

```diff
@@ -66,7 +68,155 @@ The expected output is something like the following
         },
         "NumberOfBrokerNodes": 3,
         "State": "ACTIVE",
-        "ZookeeperConnectString": "localhost:4515"
+        "ZookeeperConnectString": "localhost:4513"
     }
 }
-{{< / command >}}
\ No newline at end of file
+{{< / command >}}
+
+## Create a kafka topic
+
+In this step of using LocalStack MSK, you create a client machine. You use this client machine to create a topic that produces and consumes data.
```

Comment:
The term "client machine" is a bit misleading (not really common), can we change this to something like:

```
In this step of using LocalStack MSK, we'll download and use the Kafka command line interface (CLI) to create a topic that produces and consumes data.
```

Reply:
Done